app/models/edition.rb

```diff
@@ -87,6 +88,14 @@ def base_path_present?
     base_path.present?
   end
 
+  def copy_links_from(other_links)
```

Comment:
Harking back to the original big object avoiding intent of the 'pillars', should this be a concern of `Edition`? As we've got command objects doing things to editions would this be better in there or in its own class? It looks like it's being tested within the `put_content` command tests already.